linters/importorder/README.md

```diff
@@ -0,0 +1,79 @@
+# ImportOrder
+
+ImportOrder is a Golang linter that detects misordered imports given a certain pattern set.
+
+For example, let's say you run the following `importorder` command:
+
+./importorder -patterns="STDLIB github.com/m3db/m3coordinator github.com/m3db EXTERNAL" path/to/directory
+
+This will make sure that your import groups (in `path/to/directory`) start with all Go standard library imports, followed by imports in the package github.com/m3db/m3coordinator, followed by all imports in the github.com/m3db package, and finally all external/third party packages.
+
+In other words, this block will succeed:
+```
+import (
+	"context"
+	"time"
+
+	"github.com/m3db/m3coordinator/errors"
+	"github.com/m3db/m3coordinator/policy/resolver"
+
+	"github.com/m3db/m3db/client"
+	"github.com/m3db/m3x/ident"
+)
+```
+
+And this will fail because `context` should belong in the first group with the other standard library import:
+```
+import (
+	"time"
+
+	"github.com/m3db/m3coordinator/errors"
+  "context"
```

Comment:
nit: could you align `context` correctly with the other imports in it's group

Reply:
Hmm that's weird- it shows correctly on my laptop.